pkg/nuclide/file-tree/spec/FileTreeController-spec.js

```diff
@@ -66,6 +66,21 @@ describe('FileTreeController', () => {
     fileTreeController.destroy();
   });
 
+  it('checks if core:backspace and core:delete is registered on .nuclide-file-tree', () => {
```

Comment:
Thanks for adding `core:backspace` too. Can you split this into two specs? It might feel like duplication, but it'll be more instructive in the future if one of these commands break because a single, targeted spec will say exactly what failed.